### DISCUSSION TITLE
When?

### DISCUSSION THREAD

**Discussion by determin1st** [mishmanners/GitHub-Like-A-Boss#11] [General] [answered]
So when will this feature become available?
Maybe im blind and it's already there somewhere?

**Comment by mishmanners** [comment]
Which feature? You mean Codespaces?

**Comment by determin1st** [comment]
![EXBKLuYWsAA4vGK jpg large - копия - копия](https://user-images.githubusercontent.com/16524081/86134725-66aaff80-bb03-11ea-8043-b8b57fa207c6.jpg)

**Comment by mishmanners** [comment, marked as answer]
Oh Discussions... yep coming very soon. They said the middle of the year, so should be VERY soon 😄
